Modules/cjkcodecs/multibytecodec.c

```diff
@@ -979,8 +1005,16 @@ _multibytecodec_MultibyteIncrementalEncoder_setstate_impl(MultibyteIncrementalEn
     }
 
     if (statebytes[0] > MAXENCPENDING*4) {
-        PyErr_SetString(PyExc_UnicodeError, "pending buffer too large");
-        return NULL;
+        PyObject *excobj = PyObject_CallFunction(PyExc_UnicodeEncodeError,
+                                                 "ssnns",
+                                                 self->codec->encoding,
+                                                 statebytes,
+                                                 0, sizeof(statebytes),
+                                                 "pending buffer too large");
+        if (excobj == NULL) goto errorexit;
+        PyErr_SetObject(PyExc_UnicodeEncodeError, excobj);
```

Comment:
I don't feel this is valid UnicodeEncodeError.
Should we really avoid UnicodeError here?

Reply:
If `UnicodeEncodeError` is because the user passed an input that directly causes the encoding problem, rather than any problem that occurs during encoding, then `UnicodeError` makes sense here. Changed this one back to `UnicodeError`.